packages/vuetify/src/components/VAlert/VAlert.tsx

```diff
@@ -0,0 +1,200 @@
+// Styles
+import './VAlert.sass'
+
+// Components
+import { VAvatar } from '@/components/VAvatar'
+import { VBtn } from '@/components/VBtn'
+
+// Composables
+import { makeDensityProps, useDensity } from '@/composables/density'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { makeRoundedProps, useRounded } from '@/composables/rounded'
+import { makeTagProps } from '@/composables/tag'
+import { makeThemeProps, useTheme } from '@/composables/theme'
+import { makeVariantProps, useVariant } from '@/composables/variant'
+import { useBorder } from '@/composables/border'
+import { useProxiedModel } from '@/composables/proxiedModel'
+import { useTextColor } from '@/composables/color'
+
+// Utilities
+import { computed, defineComponent } from 'vue'
+import { makeProps } from '@/util'
+
+// Types
+import type { PropType } from 'vue'
+
+export const allowedTypes = ['success', 'info', 'warning', 'error'] as const
+
+export type Type = typeof allowedTypes[number]
```

Comment:
not sure about the name, it's too generic imho
also it would be used for snackbars as well, so it could be defined somewhere outside, i don't have a good place for this though since `vuetify/types` are gone (temporarily?)

Reply:
The name of the type or the name `type` itself for the prop?

Reply:
the name of the type